ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/UnsignedLongParameter.java

```diff
@@ -24,7 +24,7 @@ public class UnsignedLongParameter {
 
   @JsonCreator
   public UnsignedLongParameter(final String value) {
-    this.value = Long.decode(value);
+    this.value = value != null ? Long.decode(value) : 0;
```

Comment:
Is 0 a safe default for all UnsignedLongParameters?

Reply:
instead of supporting null, what if introducing an `optionalUnsignedLongParameters`?

Reply:
I'm not sure. I'm more sure that 'null' is unsafe... would you prefer 0, null, or a thrown exception of some sort here? Other options?

Reply:
I guess I don't understand the impact on any existing UnsignedLongParameter value. If there is no impact, i.e. no bug introduced by this, then I'm easy with any of those. Exception might be more explicit and good 'fail early' practice though.

Reply:
I agree with the backward compatibility, and looking at similar classes this will bit a bit odd, since none is supporting null, so we probably need to find another way to reach the same goal

Reply:
Turns out this is only ever used by test code, and never passes in null, so this is fine to revert.